http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -274,6 +274,14 @@ object Config {
     )
   }
 
+  implicit val persistentDiskConfigReader: ValueReader[PersistentDiskConfig] = ValueReader.relative { config =>
```

Comment:
make this private...we should use this reader outside this file

Reply:
should I go ahead and make all of the readers here private then?

Reply:
nah...I did that in my other PR...

Reply:
making this private made it inaccessible for the configs in `CommonTestData` - did you move those around in your PR as well?

Reply:
yea that's exactly the point...we shouldn't use it in `CommonTestData`...you can use `Config.persistentDisk` (whatever name you give it) in test the same way you do in app code...That way, our unit tests will actually be testing the Config code as well instead of doing its own thing in test